src/GovUk.Education.ExploreEducationStatistics.Data.Api/Services/DataService.cs

```diff
@@ -36,5 +37,21 @@ public override ResultWithMetaViewModel Query(ObservationQueryContext queryConte
                 MetaData = _subjectMetaService.GetSubjectMeta(queryContext.ToSubjectMetaQueryContext(), observations)
             };
         }
+        
+        public override async Task<ResultWithMetaViewModel> QueryAsync(ObservationQueryContext queryContext)
+        {
```

Comment:
Seems to be identical to the `Query` method, just with an async, which is duplicating all the existing code.

Reply:
I've amended this to just call the none async method, rather than duplicate. This allows us to call the query with an await in another service (as done in this PR) even though the query method its self isnt async.